### DISCUSSION TITLE
Algum Participante para fazer testes?

### DISCUSSION THREAD

**Discussion by jeanhora** [bacen/pix-api#431] [General]
Algum Participante para fazer testes entre psps, envio de pacs008 e pacs004 no ambiente de homologação ?

**Comment by MAKoprovski** [comment]
Olá @jeanhora,
Nós estamos abertos para lhe auxiliar neste processo. Estamos buscando interessados em participar.
Para maiores detalhes entre em contato via email makoprovski@gmail.com
